src/neo/NeoSystem.cs

```diff
@@ -86,27 +87,28 @@ public class NeoSystem : IDisposable
         internal RelayCache RelayCache { get; } = new(100);
 
         private ImmutableList<object> services = ImmutableList<object>.Empty;
+        private ImmutableList<Plugin> plugins = ImmutableList<Plugin>.Empty;
         private readonly string storage_engine;
         private readonly IStore store;
         private ChannelsConfig start_message = null;
         private int suspend = 0;
 
-        static NeoSystem()
-        {
-            // Unify unhandled exceptions
-            AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
-
-            Plugin.LoadPlugins();
-        }
-
         /// <summary>
         /// Initializes a new instance of the <see cref="NeoSystem"/> class.
         /// </summary>
         /// <param name="settings">The protocol settings of the <see cref="NeoSystem"/>.</param>
         /// <param name="storageEngine">The storage engine used to create the <see cref="IStore"/> objects. If this parameter is <see langword="null"/>, a default in-memory storage engine will be used.</param>
         /// <param name="storagePath">The path of the storage. If <paramref name="storageEngine"/> is the default in-memory storage engine, this parameter is ignored.</param>
-        public NeoSystem(ProtocolSettings settings, string storageEngine = null, string storagePath = null)
+        /// <param name="plugins">Collection of Plugins to preload into NeoSystem instance</param>
+        public NeoSystem(ProtocolSettings settings, string storageEngine = null, string storagePath = null, IEnumerable<Plugin> plugins = null)
         {
+            this.plugins = plugins == null ? ImmutableList<Plugin>.Empty : plugins.ToImmutableList();
+
+            var logger = ActorSystem.ActorOf<Logger>();
```

Comment:
You create a new `Logger` instance here. But `ActorSystem` will use the automatically created `Logger` instance to write the logs, and it will not be passed into the `SetNeoSystem` object.

Reply:
What is the correct way to get an existing actor already created by the system? Unfortunately, there doesn't appear to be a way to provide parameters or used Props based actor construction for logger actors

Reply:
I don't know any good way. That's why I didn't put all the plug-in instances in `NeoSystem`.

Reply:
`ActorSystem.ActorSelection("/system/log*").Tell` sends a message to all loggers registered in the system. I can use that

Reply:
Do we have multiple loggers in system? How about ActorSelection => Actorof?